src/ContextMenu.tsx

```diff
@@ -132,14 +140,46 @@ export const ContextMenuItem: FunctionComponent<IContextMenuItem> = props => {
           data={data}
           preventClose={preventClose}
           disabled={disabled}
-          onClick={onClick}
+          onClick={(e, d, t) => {
+            if (onClick) {
+              return onClick(e, d, t);
+            }
+          }}
         >
-          {title && <Text>{title}</Text>}
-          {divider && <Break thickness={1} />}
+          <Flex alignItems="center">
+            {title ? <Text flex={1}>{title}</Text> : null}
+            {isSubMenu ? <Text pl="5px">&#9658;</Text> : null}
+          </Flex>
+
+          {divider ? <Break thickness={1} /> : null}
         </ReactMenuItem>
       )}
     />
   );
+
+  if (isSubMenu) {
+    return (
+      <Box
+        {...rest}
+        css={menuStyles()}
+        as={({ className }: { className: string }) => {
```

Comment:
I'd move it to a separate component.
Look at Input for a reference.
We could also make use of [Emotion's ClassNames](https://emotion.sh/docs/class-names).

Reply:
Disagree with moving since it heavily depends on menu item. Cannot currently think of a use case where it doesn't. Also cannot think of a use case where we would need to only import submenu, leaving as it for and can update later if needed

Reply:
Don't remember the case, but I used to have troubles with such an approach.
Basically, in this case, you are kind of creating a new component every single time.
I am fine with leaving it as is given it works as expected.